tracing/opentracing/server_interceptors.go

```diff
@@ -28,7 +28,7 @@ func UnaryServerInterceptor(opts ...Option) grpc.UnaryServerInterceptor {
 		}
 		newCtx, serverSpan := newServerSpanFromInbound(ctx, o.tracer, info.FullMethod)
 		resp, err := handler(newCtx, req)
-		finishServerSpan(ctx, serverSpan, err)
+		finishServerSpan(ctx, serverSpan, err, false)
```

Comment:
Lets pass this via `opts` ... maybe something like `DisableErrorTracing` default to true so not to change behaviour